e2e/tests/getDefinitionAtPosition/test.js

```diff
@@ -0,0 +1,49 @@
+// @ts-check
+const path = require('path');
+const assert = require('chai').assert;
+
+const createServer = require('../../_server');
+const { openMockFile, getFirstResponseOfType, getResponsesOfType } = require('../../_helpers');
+
+const mockFileName = 'main.ts';
+
+const createServerWithMockFile = (fileContents) => {
+    const server = createServer(__dirname, 'echo-plugin');
+    openMockFile(server, mockFileName, fileContents);
+    return server;
+}
+
+describe('GetDefinitionAtPosition', () => {
+    it('should return blank definition at template literal position', async () => {
+        return getDefinitionAtPositionInMockFile(
+            'const q = test`abcdefg`',
+            { offset: 16, line: 1 },
+        ).then(server => {
+            const definitions = getResponsesOfType('goToDefinition', server);
+            // Default goToDefinition inside tagged template literal should be blank
+            assert.strictEqual(definitions.length, 0);
+        })
+    });
+
+    it('should return definition at position', async () => {
+        return getDefinitionAtPositionInMockFile(
+            `const a = 'test';
+console.log(a);`,
+            { offset: 12, line: 1 },
```

Comment:
Line and column start at one, not zero. Try: line 2, offset 13 (or maybe 14)

Reply:
I tried the suggested values already, but no luck.

Reply:
Let me make the variable multi-character and choose an index in the middle.

Reply:
Nope, no luck even with that, anyways pushing it as it is less ambiguous!